metricbeat/module/vsphere/virtualmachine/virtualmachine.go

```diff
@@ -215,8 +215,20 @@ func getNetworkNames(c *vim25.Client, ref types.ManagedObjectReference) ([]strin
 		return nil, errors.Wrap(err, "no networks found")
 	}
 
+	networksRefs := []types.ManagedObjectReference{}
```

Comment:
Use `var networkRefs types.ManagedObjectReferenc` instead. https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

Reply:
Changed, thanks for the link!